index.css

```diff
@@ -0,0 +1,154 @@
+@media (min-width: 400px) {
```

Comment:
Please don't use use media queries yet. We are using mobile to desktop approach right now. You can use them on the desktop version on the future lessons. And please check Figma template below, the width here is 375px. Kindly remove media queries.
<img width="412" alt="Screenshot 2022-12-02 at 2 14 22 AM" src="https://user-images.githubusercontent.com/25887253/205178962-93804014-04c0-4a63-b550-b656838d1416.png">